php/README.md

```diff
@@ -40,16 +46,12 @@ INSTALL
  - (Temporary workaround) Compile gRPC extension from source
 
    ```
+   $ export LINUXBREWHOME="$HOME/.linuxbrew"
```

Comment:
I think
```sh  
export BREWHOME=`brew config | grep HOMEBREW_PREFIX | cut -f 2 -d'  '`
```
works on both mac and linux, but might be a bit of handle to copy and paste.

It might be worth it to add a script to this folder combines this and the next few lines.